auth-web/src/util/constants.ts

```diff
@@ -114,6 +114,7 @@ export enum AccountStatus {
     PENDING_STAFF_REVIEW = 'PENDING_STAFF_REVIEW',
     PENDING_ACTIVATION = 'PENDING_ACTIVATION',
     NSF_SUSPENDED = 'NSF_SUSPENDED',
+    EFT_SUSPENDED = 'EFT_SUSPENDED',
```

Comment:
we're not adding in another code.. 

You'll have to read from the suspension reason code to determine if it's NSF or EFT NSF 

![image](https://github.com/bcgov/sbc-auth/assets/3484109/0aa227f9-5595-47bf-a9ca-234578fb3dc4)

Reply:
talk to @Jxio about this one

Reply:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/1156a519-2ac8-44ec-ad58-57b997acea99)

Reply:
@Jxio we'll need a status for PAD NSF

Reply:
we probably need to add in the PAD NSF one here:
https://github.com/bcgov/sbc-pay/blob/39f4ef79c2d0f4de56c444406166cb723dfb546f/pay-queue/src/pay_queue/services/payment_reconciliations.py#L690

Reply:
@Jxio the problem with using suspended though, there's an entire flow in sbc-common-components that looks for nsf_suspended and not suspended?

Reply:
https://github.com/bcgov/sbc-common-components/blob/94986110a7f6c7ba4f57ed8b038101ba7d864a94/vue/sbc-common-components/src/components/SbcAuthMenu.vue#L159

Reply:
https://github.com/bcgov/sbc-common-components/blob/94986110a7f6c7ba4f57ed8b038101ba7d864a94/vue/sbc-common-components/src/components/SbcHeader.vue#L587

Reply:
I think we only used nsf_suspended for eft currently.

Reply:
yeah maybe just use `nsf_suspended` then, we'll need another field in the `suspension_reason_codes` to differentiate between PAD NSF and EFT NSF?